Tests/PostgresNIOTests/New/Messages/AuthenticationTests.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  File.swift
+//  
+//
+//  Created by Fabian Fett on 07.01.21.
+//
+
+import NIO
```

Comment:
Ditch Xcode template headers.
```suggestion
import NIO
```